scripts/change-log-constants.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
```

Comment:
In general, replace the use of var for let or const. I think we might need to update our lint config to cover the scripts.